sklearn/linear_model/perceptron.py

```diff
@@ -115,6 +115,10 @@ class Perceptron(BaseSGDClassifier):
         Number of weight updates performed during training.
         Same as ``(n_iter_ * n_samples)``.
 
+    loss_function_ : concrete ``LossFunction`` 
```

Comment:
we don't use backticks for the they, I don't think it will render properly

Reply:
this was copied from SGDClassifier